client/src/components/CitizenRequest/CitizenRequest.component.tsx

```diff
@@ -36,7 +37,7 @@ const CitizenRequest = ({ agency }: { agency: Agency }): JSX.Element => {
     captchaResponse: string,
   ): Promise<void> => {
     const mail: Mail = {
-      agencyId: agency.id ? [agency.id] : [],
+      agencyId: agency?.id ? [agency?.id] : [],
```

Comment:
is there a possibility of the `id` being 0, which would make the boolean check unintentionally falsy?